### PR TITLE
Add UI Tests

### PR DESCRIPTION
**Pull request by sakshi173**
GitHub Issue (If applicable): #

## PR Type

What kind of change does this PR introduce?
- Other... Please describe:
    - UI Test

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tested on iOS.
- [X] Tested on Wasm.
- [X] Tested on Android.
- [ ] Tested on UWP.
- [ ] Tested in both **Light** and **Dark** themes.
- [ ] Associated with an issue (GitHub or internal)

### CODE REVIEW

File: Uno.Gallery/Uno.Gallery.UITest/Constants.cs [hunk 1]
```diff
@@ -9,7 +9,7 @@ namespace Uno.Gallery.UITests
 {
     public class Constants
     {
-        public readonly static string WebAssemblyDefaultUri = "http://localhost:64555/";
+        public readonly static string WebAssemblyDefaultUri = "http://localhost:56745/";
```
Comment (carldebilly) [inline Uno.Gallery/Uno.Gallery.UITest/Constants.cs:12]: The port should be fixed using a `launchSettings.json` file [like we did on the sample app](https://github.com/unoplatform/uno/blob/master/src/SamplesApp/SamplesApp.Wasm/Properties/launchSettings.json).

Comment (kazo0) [reply Uno.Gallery/Uno.Gallery.UITest/Constants.cs:12]: There's still a port define in the [Constants.cs](https://github.com/unoplatform/uno/blob/master/src/SamplesApp/SamplesApp.UITests/Constants.cs) no?

**Review by kazo0 — APPROVED** [review]

File: Uno.Gallery/Uno.Gallery.UITest/Constants.cs (hunk 1 above)
Comment (carldebilly) [reply Uno.Gallery/Uno.Gallery.UITest/Constants.cs:12]: Yes... but why are you changing it?

Comment (kazo0) [reply Uno.Gallery/Uno.Gallery.UITest/Constants.cs:12]: It didn't match the port in the launchSettings and was not launching the tests for WASM properly

**Pull request merged**